### PR TITLE
Tests and upper-/lowercase support

### LINKED ISSUE #1: Support lowercase LCID
While LCID usually have language in lower case and country in uppercase it would be nice if the lib also supported passing in both in lower case.

see also https://github.com/ember-intl/ember-intl/pull/438

### PR DESCRIPTION
**Pull request by Turbo87**
This PR adds a few basic unit tests using mochajs.org and support for lowercase and uppercase LCIDs.

Resolves #1 

/cc @10xjs

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    'sk_Latin_SK': '🇸🇰',
+  };
+
+  Object.keys(tests).forEach(function(from) {
+    var to = tests[from];
+    it('converts "' + from + '" to ' + to, function() {
+      expect(localeEmoji(from)).to.equal(to);
+    });
+  });
+});
```
Comment (10xjs) [inline test.js]: I'm not a huge fan of dynamic tests.

Make any `if` declaration static.  The `expect` inside a `foreach` is 👌 tho.

Also, trailing line break.

**Review by 10xjs — CHANGES_REQUESTED** [review]
Changes look good!

A few nits on the tests.

**Comment by 10xjs** [comment]
@Turbo87 I'll get CI set up if you want to add a `.travis.yml` to this PR.

```yml
sudo: false
language: node_js
matrix:
  include:
    - node_js: 4
    - node_js: 5
    - node_js: 6
```

**Comment by Turbo87** [comment]
@10xjs thanks for the quick review. I've addressed your comments and added a TravisCI config file.

**Review by 10xjs — APPROVED** [review]

**Pull request merged**